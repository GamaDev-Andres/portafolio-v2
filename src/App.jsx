import { useEffect } from 'react'
import Footer from './components/Footer'

import AppRouter from './routes/AppRouter'
import { HelmetProvider } from 'react-helmet-async'

const App = () => {
  useEffect(() => {
    const isDark = localStorage.getItem('isDark')
    if (isDark === 'true') {
      document.documentElement.classList.add('dark')
    }
  }, [])

  return (
    <HelmetProvider>
      <div className="bg-base dark:bg-baseDark transition-colors duration-150 text-letra dark:text-letraDark min-w-screen">
        <div className="grid grid-rows-layout grid-cols-1 max-w-[1256px] min-h-screen mx-auto">
          <AppRouter />
          <Footer />
        </div>
      </div>
    </HelmetProvider>
  )
}

export default App
