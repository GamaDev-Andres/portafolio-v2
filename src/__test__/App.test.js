/**
 * @jest-environment jsdom
 */
import Enzyme from "enzyme";
import React from "react";
import App from "../App";

describe("probando <App/>", () => {
  test("debe montarse correctamente", () => {
    const wrapper = Enzyme.mount(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
