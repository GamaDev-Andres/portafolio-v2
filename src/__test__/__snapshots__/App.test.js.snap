// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`probando <App/> debe montarse correctamente 1`] = `
<div
  className="h-screen flex justify-center items-center bg-blue-500"
>
  <h1
    className="p-8 rounded-2xl text-white text-5xl font-bold  bg-violet-900"
  >
    Iniciando portafolio-v2
  </h1>
</div>
`;
